web/models/card.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Plixir.Card do
+  @doc """
+  Given a string card, returns its integer value
+
+  ## Examples
+
+    iex> Plixir.Card.value("AC")
+    14
+    iex> Plixir.Card.value("QD")
+    11
+    iex> Plixir.Card.value("6S")
+    6
+
+  """
+  def value("A" <> _) do 14 end
```

Comment:
if it's a one-line function, I do:
```elixir
def value("A" <> _), do: 14
```

but I'm not sure what it's the convention